src/OrangePaymentSlipData.php

```diff
@@ -100,6 +108,7 @@ public function setWithBankingCustomerId($withBankingCustomerId = true)
 
         if ($withBankingCustomerId === false) {
             $this->bankingCustomerId = '';
+            $this->bankingCustomerIdLength = 0;
```

Comment:
Setting it to zero seems to be a bit silly. Either unset or a reasonable default.